python/FileIO.py

```diff
@@ -8,4 +8,7 @@ def append(args):
 
 def read(args):
 	with open(str(args[0])[1:-1], "r", encoding="utf-8") as f:
-		return f.read()
\ No newline at end of file
+		return f.read()
+
+def _values():
+	return {"write": None, "append": None, "read": None}
```

Comment:
The type annotations should also have the argument types so that they, too, can be type checked